build_and_exec.sh

```diff
@@ -0,0 +1,7 @@
+# format of tag: 01-01-2020t12.00pm (lowercase t, am, pm)
```

Comment:
We don't seem to have a reference to this file anywhere. If we're going to include it for ease of running, let's add a call out in the README so we know what it is used for. In the past there's been some random scripts in the ODE and its submodules that we haven't been able to easily determine their purpose and we should avoid that in the future.

Reply:
Sounds good, I've added a section on the `build_and_exec.sh` script to the README.